Assets/Scripts/Game Scene/GameOverHUDController.cs

```diff
@@ -0,0 +1,25 @@
+using TMPro;
+using UnityEngine;
+
+public class GameOverHUDController : MonoBehaviour
+{  
+    private Transform gameOverPanel;
+    private Transform highscoreMessage;
+
+    private void Start()
+    {
+        gameOverPanel = gameObject.transform.Find("GameOverPanel");
+        highscoreMessage = gameOverPanel.transform.Find("HighscoreMessage");
+        highscoreMessage?.gameObject.SetActive(false);
+    }
```

Comment:
As far as I can remember `.Find` is kinda expensive. Isn't better to make those Transforms `SerializeField` and put them manually on the inspector?